{"uid":"1f48afff9c5ce1d3","name":"test_tls_enabled","fullName":"tests.integration.test_tls#test_tls_enabled","historyId":"0ebe5aa0b66f6c5314fa764ad3239c77","time":{"start":1725154679867,"stop":1725155528597,"duration":848730},"description":"Test that TLS is enabled when relating to the TLS Certificates Operator.","descriptionHtml":"<p>Test that TLS is enabled when relating to the TLS Certificates Operator.</p>\n","status":"broken","statusMessage":"tenacity.RetryError: RetryError[<Future at 0x7f065248bfa0 state=finished returned bool>]","statusTrace":"Traceback (most recent call last):\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 341, in from_call\n    result: Optional[TResult] = func()\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 241, in <lambda>\n    lambda: runtest_hook(item=item, **kwds), when=when, reraise=reraise\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_hooks.py\", line 513, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 182, in _multicall\n    return outcome.get_result()\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_result.py\", line 100, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/threadexception.py\", line 87, in pytest_runtest_call\n    yield from thread_exception_runtest_hook()\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/threadexception.py\", line 63, in thread_exception_runtest_hook\n    yield\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/unraisableexception.py\", line 90, in pytest_runtest_call\n    yield from unraisable_exception_runtest_hook()\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/unraisableexception.py\", line 65, in unraisable_exception_runtest_hook\n    yield\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/logging.py\", line 850, in pytest_runtest_call\n    yield from self._runtest_for(item, \"call\")\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/logging.py\", line 833, in _runtest_for\n    yield\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/capture.py\", line 878, in pytest_runtest_call\n    return (yield)\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/skipping.py\", line 257, in pytest_runtest_call\n    return (yield)\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 183, in pytest_runtest_call\n    raise e\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 173, in pytest_runtest_call\n    item.runtest()\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/python.py\", line 1632, in runtest\n    self.ihook.pytest_pyfunc_call(pyfuncitem=self)\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_hooks.py\", line 513, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 182, in _multicall\n    return outcome.get_result()\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_result.py\", line 100, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/python.py\", line 162, in pytest_pyfunc_call\n    result = testfunction(**testargs)\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pytest_asyncio/plugin.py\", line 529, in inner\n    _loop.run_until_complete(task)\n  File \"/usr/lib/python3.10/asyncio/base_events.py\", line 649, in run_until_complete\n    return future.result()\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/tests/integration/test_tls.py\", line 156, in test_tls_enabled\n    assert await primary_changed(ops_test, primary), \"primary not changed\"\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/tenacity/asyncio/__init__.py\", line 189, in async_wrapped\n    return await copy(fn, *args, **kwargs)\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/tenacity/asyncio/__init__.py\", line 111, in __call__\n    do = await self.iter(retry_state=retry_state)\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/tenacity/asyncio/__init__.py\", line 153, in iter\n    result = await action(retry_state)\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/tenacity/_utils.py\", line 99, in inner\n    return call(*args, **kwargs)\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/tenacity/__init__.py\", line 419, in exc_check\n    raise retry_exc from fut.exception()\ntenacity.RetryError: RetryError[<Future at 0x7f065248bfa0 state=finished returned bool>]","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":1,"retriesStatusChange":true,"beforeStages":[{"name":"event_loop","time":{"start":1725154672050,"stop":1725154672050,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"tmp_path_factory","time":{"start":1725154672050,"stop":1725154672050,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"ops_test","time":{"start":1725154672051,"stop":1725154672200,"duration":149},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"abort_on_fail","time":{"start":1725154679867,"stop":1725154679867,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"testStage":{"description":"Test that TLS is enabled when relating to the TLS Certificates Operator.","status":"broken","statusMessage":"tenacity.RetryError: RetryError[<Future at 0x7f065248bfa0 state=finished returned bool>]","statusTrace":"Traceback (most recent call last):\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 341, in from_call\n    result: Optional[TResult] = func()\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 241, in <lambda>\n    lambda: runtest_hook(item=item, **kwds), when=when, reraise=reraise\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_hooks.py\", line 513, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 182, in _multicall\n    return outcome.get_result()\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_result.py\", line 100, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/threadexception.py\", line 87, in pytest_runtest_call\n    yield from thread_exception_runtest_hook()\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/threadexception.py\", line 63, in thread_exception_runtest_hook\n    yield\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/unraisableexception.py\", line 90, in pytest_runtest_call\n    yield from unraisable_exception_runtest_hook()\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/unraisableexception.py\", line 65, in unraisable_exception_runtest_hook\n    yield\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/logging.py\", line 850, in pytest_runtest_call\n    yield from self._runtest_for(item, \"call\")\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/logging.py\", line 833, in _runtest_for\n    yield\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/capture.py\", line 878, in pytest_runtest_call\n    return (yield)\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/skipping.py\", line 257, in pytest_runtest_call\n    return (yield)\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 183, in pytest_runtest_call\n    raise e\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 173, in pytest_runtest_call\n    item.runtest()\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/python.py\", line 1632, in runtest\n    self.ihook.pytest_pyfunc_call(pyfuncitem=self)\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_hooks.py\", line 513, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 182, in _multicall\n    return outcome.get_result()\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_result.py\", line 100, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/python.py\", line 162, in pytest_pyfunc_call\n    result = testfunction(**testargs)\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pytest_asyncio/plugin.py\", line 529, in inner\n    _loop.run_until_complete(task)\n  File \"/usr/lib/python3.10/asyncio/base_events.py\", line 649, in run_until_complete\n    return future.result()\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/tests/integration/test_tls.py\", line 156, in test_tls_enabled\n    assert await primary_changed(ops_test, primary), \"primary not changed\"\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/tenacity/asyncio/__init__.py\", line 189, in async_wrapped\n    return await copy(fn, *args, **kwargs)\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/tenacity/asyncio/__init__.py\", line 111, in __call__\n    do = await self.iter(retry_state=retry_state)\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/tenacity/asyncio/__init__.py\", line 153, in iter\n    result = await action(retry_state)\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/tenacity/_utils.py\", line 99, in inner\n    return call(*args, **kwargs)\n  File \"/home/runner/work/test-runners-2-github-x64-postgresql-operator/test-runners-2-github-x64-postgresql-operator/.tox/integration/lib/python3.10/site-packages/tenacity/__init__.py\", line 419, in exc_check\n    raise retry_exc from fut.exception()\ntenacity.RetryError: RetryError[<Future at 0x7f065248bfa0 state=finished returned bool>]","steps":[],"attachments":[{"uid":"49dc54dbf5c33bc4","name":"log","source":"49dc54dbf5c33bc4.txt","type":"text/plain","size":5696}],"parameters":[],"stepsCount":0,"shouldDisplayMessage":true,"attachmentsCount":1,"hasContent":true,"attachmentStep":false},"afterStages":[{"name":"abort_on_fail::0","time":{"start":1725155528600,"stop":1725155528600,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"ops_test::finalizer","time":{"start":1725155528602,"stop":1725155528905,"duration":303},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::2","time":{"start":1725155528906,"stop":1725155528906,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::_close_event_loop","time":{"start":1725155528906,"stop":1725155528906,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::_provide_clean_event_loop","time":{"start":1725155528906,"stop":1725155528906,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"labels":[{"name":"tag","value":"group(1)"},{"name":"tag","value":"asyncio"},{"name":"parentSuite","value":"tests.integration"},{"name":"suite","value":"test_tls"},{"name":"host","value":"fv-az891-324"},{"name":"thread","value":"9044-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.integration.test_tls"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[{"uid":"3f55c6980286bbd","status":"unknown","time":{}}],"categories":[{"name":"Test defects","matchedStatuses":[],"flaky":false}],"history":{"statistic":{"failed":0,"broken":23,"skipped":0,"passed":57,"unknown":5,"total":85},"items":[{"uid":"79775c7038a58266","reportUrl":"../159//#testresult/79775c7038a58266","status":"broken","statusDetails":"tenacity.RetryError: RetryError[<Future at 0x7f94e9c8e9e0 state=finished returned bool>]","time":{"start":1725068241111,"stop":1725069094655,"duration":853544}},{"uid":"4441540b9a8347f9","reportUrl":"../158//#testresult/4441540b9a8347f9","status":"passed","time":{"start":1724981842414,"stop":1724983307935,"duration":1465521}},{"uid":"f9518fe177915b66","reportUrl":"../157//#testresult/f9518fe177915b66","status":"passed","time":{"start":1724895383643,"stop":1724896237729,"duration":854086}},{"uid":"41c1fa42b2c8e10c","reportUrl":"../156//#testresult/41c1fa42b2c8e10c","status":"passed","time":{"start":1724809054804,"stop":1724810581833,"duration":1527029}},{"uid":"8aa0a5da4abc6d6d","reportUrl":"../155//#testresult/8aa0a5da4abc6d6d","status":"passed","time":{"start":1724722649671,"stop":1724723450750,"duration":801079}},{"uid":"6792252fa606e8a8","reportUrl":"../154//#testresult/6792252fa606e8a8","status":"passed","time":{"start":1724636203683,"stop":1724637060379,"duration":856696}},{"uid":"d6b32ee9c6c56fb0","reportUrl":"../153//#testresult/d6b32ee9c6c56fb0","status":"broken","statusDetails":"tenacity.RetryError: RetryError[<Future at 0x7f657ec11510 state=finished returned bool>]","time":{"start":1724549831853,"stop":1724550698023,"duration":866170}},{"uid":"9be4b9b0d522a1d","reportUrl":"../152//#testresult/9be4b9b0d522a1d","status":"passed","time":{"start":1724463384944,"stop":1724464321702,"duration":936758}},{"uid":"fb927e790864938f","reportUrl":"../151//#testresult/fb927e790864938f","status":"passed","time":{"start":1724376966058,"stop":1724378419245,"duration":1453187}},{"uid":"10bf27916cb39aaa","reportUrl":"../150//#testresult/10bf27916cb39aaa","status":"passed","time":{"start":1724290586954,"stop":1724291422701,"duration":835747}},{"uid":"35b58f0cee8064b7","reportUrl":"../149//#testresult/35b58f0cee8064b7","status":"broken","statusDetails":"tenacity.RetryError: RetryError[<Future at 0x7f61f7329c00 state=finished returned bool>]","time":{"start":1724204173424,"stop":1724205027056,"duration":853632}},{"uid":"9a11b216d5ddaaa3","reportUrl":"../148//#testresult/9a11b216d5ddaaa3","status":"broken","statusDetails":"tenacity.RetryError: RetryError[<Future at 0x7fc26bfaa9b0 state=finished returned bool>]","time":{"start":1724118104357,"stop":1724118946905,"duration":842548}},{"uid":"90c51de035fd290","reportUrl":"../147//#testresult/90c51de035fd290","status":"passed","time":{"start":1724031378594,"stop":1724032845331,"duration":1466737}},{"uid":"13364c96f8e8cb26","reportUrl":"../146//#testresult/13364c96f8e8cb26","status":"passed","time":{"start":1723945056476,"stop":1723945924689,"duration":868213}},{"uid":"47340851ae33585e","reportUrl":"../145//#testresult/47340851ae33585e","status":"passed","time":{"start":1723858632672,"stop":1723859455475,"duration":822803}},{"uid":"215b80086040463b","reportUrl":"../144//#testresult/215b80086040463b","status":"broken","statusDetails":"tenacity.RetryError: RetryError[<Future at 0x7f5f8b2b22c0 state=finished returned bool>]","time":{"start":1723772267514,"stop":1723773148537,"duration":881023}},{"uid":"65db25fa36cc9804","reportUrl":"../143//#testresult/65db25fa36cc9804","status":"passed","time":{"start":1723685842637,"stop":1723687330928,"duration":1488291}},{"uid":"87a697ea44ff3a38","reportUrl":"../142//#testresult/87a697ea44ff3a38","status":"passed","time":{"start":1723599685144,"stop":1723600556006,"duration":870862}},{"uid":"72593f17e874113f","reportUrl":"../141//#testresult/72593f17e874113f","status":"passed","time":{"start":1723513927568,"stop":1723514866017,"duration":938449}},{"uid":"c9463ac40beac8e9","reportUrl":"../140//#testresult/c9463ac40beac8e9","status":"passed","time":{"start":1723426841058,"stop":1723428281473,"duration":1440415}}]},"tags":["asyncio","group(1)"]},"source":"1f48afff9c5ce1d3.json","parameterValues":[]}